cli/src/cmd/forge/script/executor.rs

```diff
@@ -155,9 +155,7 @@ impl ScriptArgs {
         let env = script_config.evm_opts.evm_env().await;
 
         // The db backend that serves all the data.
-        // We only call this function twice: when executing locally and when executing the
-        // broadcastable transactions.
-        let db = script_config.backend.take().unwrap_or_else(|| {
+        let db = script_config.backend.as_ref().cloned().unwrap_or_else(|| {
```

Comment:
don't think we need to get ref first, right?
```suggestion
        let db = script_config.backend.clone().unwrap_or_else(|| {
```